azkaban-exec-server/src/main/java/azkaban/execapp/JobRunner.java

```diff
@@ -626,6 +622,14 @@ private void doRun() {
     writeStatus();
   }
 
+  private void uploadExecutableNode() {
+    try {
+      this.loader.uploadExecutableNode(this.node, this.props);
+    } catch (final ExecutorManagerException e1) {
```

Comment:
Yes indeed. Will do. And also pass the exception to the logger for a stack trace.